common/coat_of_arms/template_lists/colored_emblem_lists.txt

```diff
@@ -668,6 +668,76 @@ colored_emblem_texture_lists = {
 			100 	= "ce_frostwolf.dds"
```

Comment:
They should be moved up.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123956471-c7417680-d9bb-11eb-995e-8e68b250470c.png)

</details>

Reply:
Nevermind, wait till my answer.